File: 2833/Main.py
class Solution:
    def furthestDistanceFromOrigin(self, moves):
        mn = 0
        mx = 0
        for ch in moves:
            if ch == 'L':
                mn -= 1
                mx -= 1
            elif ch == 'R':
                mn += 1
                mx += 1
            else:
                mn -= 1
                mx += 1
        
        return max(abs(mn), abs(mx))

File: 2833/Main.cs
public class Solution {
    public int FurthestDistanceFromOrigin(string moves) {
        int mn = 0, mx = 0;
        
        foreach (char ch in moves) {
            if (ch == 'L') {
                mn -= 1;
                mx -= 1;
            } 
            else if (ch == 'R') {
                mn += 1;
                mx += 1;
            } 
            else {
                mn -= 1;
                mx += 1;
            }
        }
        
        return Math.Max(Math.Abs(mn), Math.Abs(mx));
    }
}
